### PR TITLE
fix grpcio-tools version

### PR DESCRIPTION
**Pull request by balopat**
Fixes an erroneous upgrade (https://github.com/quantumlib/Cirq/pull/3365), that should have been more conservative. 
We have to be careful with grpcio-tools upgrades as that determines the pb2 files that are generated.
Otherwise a breakage will occur when building TFQ: the `_internal_create_key` symbol doesn't exist in older versions.

### CODE REVIEW

**Comment by balopat** [comment]
Got a flake of https://github.com/quantumlib/Cirq/issues/3763 in https://github.com/quantumlib/Cirq/pull/3849/checks?check_run_id=1967506372

File: dev_tools/conf/pip-list-dev-tools.txt [hunk 1]
```diff
@@ -12,8 +12,10 @@ filelock~=3.0.12
 
 # For generating protobufs
 
-# 1.26 is the min version that has python 3.8 wheels
-grpcio-tools~=1.26
+# 1.26.0 is the min version that has python 3.8 wheels
+# however, we can't really go much higher - the protoc version needs to be <= 3.9.2
+# otherwise it will generate pb2 files that are incompatible with tensorflow quantum
+grpcio-tools~=1.26.0
```
Comment (95-martin-orion) [inline dev_tools/conf/pip-list-dev-tools.txt:18]: This seems like a roundabout way of enforcing `protoc <= 3.9.2` - is there any reason we can't just add that as a rule instead? pip-install should ensure compatibility between the selected protoc and grpcio versions.

Comment (balopat) [reply dev_tools/conf/pip-list-dev-tools.txt:18]: grpcio-tools is an easy way of installing protoc, but it is included into the binary (see this comment for example https://github.com/grpc/grpc/issues/15675#issuecomment-397038811) - so there is no way to enforce protoc with pip.

Comment (95-martin-orion) [reply dev_tools/conf/pip-list-dev-tools.txt:18]: Seems fine to me then.

**Review by 95-martin-orion — APPROVED** [review]
LGTM. Is #3763 also expected to be resolved by this?

**Comment by balopat** [comment]
> LGTM. Is #3763 also expected to be resolved by this?

Unfortunately no :( That is just a flake that goes away when you rerun your checks.

**Pull request merged**